charts/queryservice/templates/allowlist.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "wdqs.fullname" . }}-allowlist
+data:
+  allowlist-static.txt:
+    {{- .Values.allowListStatic | toYaml | indent 2 }}
```

Comment:
Wouldn't have been surprised if you'd either hardcoded the list here but I think having the list in the deploy repo is way cleaner for when we need to update it.

I could also have seen the static list in it's own configmap separate configmap rather than a key of this map. No need to do this though.